ppx_cstruct.opam

```diff
@@ -23,5 +23,6 @@ depends: [
   "ocaml-migrate-parsetree"
   "ppx_driver"    {test & >= "v0.9.0"}
   "ppx_sexp_conv" {test}
+  "cstruct-unix"  {test}
```

Comment:
This wasn't caught due to a missing `TESTS=true` in the travis.yml -- any chance you could add that so it runs on the CI as well?